src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -310,38 +300,37 @@ private static Delegate GetMetadataGetterDelegate<TValue>(ISchema schema, int co
             return getter;
         }
 
+        /// <summary>
+        /// Legacy method to get the column index.
+        /// DO NOT USE: use <see cref="GetColumnOrNull"/> instead.
```

Comment:
>DO NOT USE [](start = 12, length = 10)

I'm guessing using actual `Obsolete` on this and other code is undesirable?

Reply:
It's 450+ uses, and they cannot be all fixed mechanically with Resharper. So I opted to internal-best-friend it. The rest of the `ISchema` will be removed, but this one, I think, may stay for longer.

Actually obsolete-ing is going to be quite disruptive to the code base either...

---
In reply to: [242417023](https://github.com/dotnet/machinelearning/pull/1894#discussion_r242417023) [](ancestors = 242417023)